Dockerfile.protocgen

```diff
@@ -0,0 +1,17 @@
+# syntax=docker/dockerfile:experimental
+
+FROM tendermintdev/sdk-proto-gen:v0.2 as build
+
+RUN apk add --no-cache --update \
+  git \
+  ca-certificates \
+  nodejs
+
+WORKDIR /workspace
+
+COPY ./go.mod .
+COPY ./go.sum .
+
+RUN go mod download
+RUN go install github.com/regen-network/cosmos-proto/protoc-gen-gocosmos
+RUN npm install -g yarn
```

Comment:
why do we need this file? And why do we need to copy go.mod file?

Reply:
We don't need this file. It was mistakenly left over from another incarnation of this PR.

Reply:
Good catch!